components/friend.php

```diff
@@ -0,0 +1,337 @@
+	 *
+	 *     $ wp bp friend reject 2161 151 2121
+	 *     Success: Friendship successfully accepted.
+	 *
+	 * @alias reject_invitation
+	 */
+	public function reject( $args, $assoc_args ) {
+		foreach ( $args as $friendship_id ) {
+			if ( friends_reject_friendship( (int) $friendship_id ) ) {
+				WP_CLI::success( 'Friendship successfully rejected.' );
+			} else {
+				WP_CLI::error( 'There was a problem rejecting the friendship.' );
+			}
+		}
+	}
+
+	/**
+	 * Check whether two users are friends.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <user>
+	 * : ID of the first user. Accepts either a user_login or a numeric ID.
+	 *
+	 * <friend>
+	 * : ID of the other user. Accepts either a user_login or a numeric ID.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     $ wp bp friend check 2161 65465
+	 *     Success: Yes, they are friends.
+	 *
+	 *     $ wp bp friend see 2121 65456
+	 *     Success: Yes, they are friends.
+	 *
+	 * @alias see
+	 */
+	public function check( $args, $assoc_args ) {
+		// Members.
+		$user = $this->get_user_id_from_identifier( $args[0] );
+		$friend = $this->get_user_id_from_identifier( $args[1] );
+
+		if ( ! $user || ! $friend ) {
+			WP_CLI::error( 'No user found by that username or ID.' );
+		}
+
+		if ( friends_check_friendship( $user->ID, $friend->ID ) ) {
+			WP_CLI::success( 'Yes, they are friends.' );
+		} else {
+			WP_CLI::error( 'No, they are not friends.' );
+		}
+	}
+
+	/**
+	 * Get a list of user's friends.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <user>
+	 * : ID of the user. Accepts either a user_login or a numeric ID.
+	 *
+	 * [--fields=<fields>]
+	 * : Fields to display.
+	 *
+	 * [--format=<format>]
+	 * : Render output in a particular format.
+	 * ---
+	 * default: table
+	 * options:
+	 *   - table
+	 *   - ids
+	 *   - csv
+	 *   - count
+	 *   - haml
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     $ wp bp friend list 65465 --format=ids
+	 *     $ wp bp friend list 2422 --format=count
+	 *
+	 * @subcommand list
+	 */
+	public function _list( $args, $assoc_args ) {
+		$formatter = $this->get_formatter( $assoc_args );
+
+		$user = $this->get_user_id_from_identifier( $args[0] );
+
+		if ( ! $user ) {
+			WP_CLI::error( 'No user found by that username or ID.' );
+		}
+
+		$friends = BP_Friends_Friendship::get_friendships( $user->ID );
+
+		if ( empty( $friends ) ) {
+			WP_CLI::error( 'This member has no friends.' );
+		}
+
+		if ( 'ids' === $formatter->format ) {
+			echo implode( ' ', wp_list_pluck( $friends, 'friend_user_id' ) ); // WPCS: XSS ok.
+		} elseif ( 'count' === $formatter->format ) {
+			$formatter->display_items( $friends );
+		} else {
+			$formatter->display_items( $friends );
+		}
+	}
+
+	/**
+	 * Generate random friendships.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--count=<number>]
+	 * : How many friendships to generate.
+	 * ---
+	 * default: 100
+	 * ---
+	 *
+	 * [--initiator=<user>]
+	 * : ID of the first user. Accepts either a user_login or a numeric ID.
+	 * ---
+	 * default: Random user.
```

Comment:
wp-cli will see this as `Random user`, which is an invalid entry. Instead, add to the short description: If none is provided, a random user will be selected for each friendship.